src/consensus/cfund.cpp

```diff
@@ -416,7 +445,7 @@ void CFund::CProposal::ToJson(UniValue& ret) const {
     }
     ret.push_back(Pair("votesYes", nVotesYes));
     ret.push_back(Pair("votesNo", nVotesNo));
-    ret.push_back(Pair("votingCycle", (uint64_t)nVotingCycle));
+    ret.push_back(Pair("votingCycle", (uint64_t)std::min(nVotingCycle, Params().GetConsensus().nCyclesProposalVoting)));
```

Comment:
```suggestion
    ret.push_back(Pair("votingCycle", (uint64_t)std::min(nVotingCycle, Params().GetConsensus().nCyclesProposalVoting))); // We return the smaller of the current voting cycle or the max voting cycle possible so we don't return the mysterious 7th voting cycle
```

Reply:
We should have a descriptive comment in this vein for this line of code and the similar one for payment requests